webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report_list_reports.js

```diff
@@ -66,10 +66,6 @@ var Reports = (function() {
        activate: function(url) {
            url = url || null;
            if (loaded) return;
-           if (Columns.value() === null) {
-               // the storage hasn't returned yet!
-               return Columns.load(Reports.activate);
-           }
```

Comment:
That's why we do code reviews :)
Again :)

I didn't mean to delete all of it. It used to be that `Columns.load()` was async so it had to be self-recursive. Now that it's sync, I can just call `Columns.load()` instead of this "Is it ready? No? Ok, then re-run".